lib/menu.js

```diff
@@ -26,6 +28,32 @@ module.exports = function (app) {
               }
             })
           }
+        },
+        {
+          label: 'Export as GeoJSON..',
+          click: function (item, focusedWindow) {
+            dialog.showSaveDialog({
+              title: 'Guardar como..',
+              defaultPath: 'export.geojson',
+              filters: [{name: 'GeoJSON', extensions: ['geojson']}]
+            }, function (filename) {
+              if (!filename) return
+              var bbox = [-Infinity, -Infinity, Infinity, Infinity]
+              var out = fs.createWriteStream(filename)
+              var ex = exportGeoJson(app.osm, {bbox: bbox})
```

Comment:
We need to pass through the presets (map) and polygon features options like we do in server.js